README.md

```diff
@@ -1 +1,3 @@
 # cs124-lab1
+
+Note to prof/ grader: I was pushing all my commits into main (instad of starter), so I created the starter branch after my last commit (and push) in main. Since starter needs to be ahead of main in order to make a pull request, I'm making this change in the README file. At the time of creating starter branch, I have pushed all my commits necessary for the lab assignment; this md edit is the only commit that is ahead of main.
```

Comment:
You were supposed to be pushing all your commits into main.  The starter branch should have referred to the first commit.